src/v2/Components/FollowButton/Button.tsx

```diff
@@ -40,12 +40,23 @@ export class FollowButton extends React.Component<Props, State> {
 
     return (
       <Button
-        longestText="following"
```

Comment:
Is there some kind of associated prop API code that needs to be removed as well?